<ui version="4.0" >
 <class>IvaPanel</class>
 <widget class="QWidget" name="IvaPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>193</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Inicial:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_Inicial</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QDoubleSpinBox" name="iva_Inicial" >
     <property name="locale" >
      <locale country="UnitedStates" language="English" />
     </property>
     <property name="accelerated" >
      <bool>true</bool>
     </property>
     <property name="correctionMode" >
      <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
     </property>
     <property name="prefix" >
      <string/>
     </property>
     <property name="suffix" >
      <string/>
     </property>
     <property name="maximum" >
      <double>99999.990000000005239</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Desconto:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_DescP</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="iva_DescP" />
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="label_8" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>%</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_Desc</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLineEdit" name="iva_Desc" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>- Desconto:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_DescT</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QLineEdit" name="iva_DescT" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Iva:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_IvaP</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="iva_IvaP" >
     <property name="editable" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>3</number>
     </property>
     <item>
      <property name="text" >
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>20</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="label_9" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>%</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_Iva</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QLineEdit" name="iva_Iva" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>+ Iva:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_IvaT</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="QLineEdit" name="iva_IvaT" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Lucro:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_LucroP</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QSpinBox" name="iva_LucroP" >
     <property name="maximum" >
      <number>100</number>
     </property>
     <property name="singleStep" >
      <number>5</number>
     </property>
     <property name="value" >
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QLabel" name="label_10" >
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>118</red>
           <green>116</green>
           <blue>108</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string>%</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_Lucro</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="3" >
    <widget class="QLineEdit" name="iva_Lucro" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Final</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>iva_Final</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3" >
    <widget class="QLineEdit" name="iva_Final" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>iva_Inicial</sender>
   <signal>valueChanged(double)</signal>
   <receiver>IvaPanel</receiver>
   <slot>doCalc()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>104</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel" >
     <x>8</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iva_DescP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IvaPanel</receiver>
   <slot>doCalc()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>88</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>5</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iva_IvaP</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>IvaPanel</receiver>
   <slot>doCalc()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>6</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>iva_LucroP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>IvaPanel</receiver>
   <slot>doCalc()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>6</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doCalc()</slot>
 </slots>
</ui>
